src/bindings/rust/elektra/src/key.rs

```diff
@@ -179,14 +197,21 @@ impl<'a> StringKey<'a> {
     }
 
     /// Returns a deep copy of the key.
-    pub fn duplicate<'b>(&'a self) -> StringKey<'b> {
-        let dup_ptr = unsafe { elektra_sys::keyDup(self.as_ref()) };
+    pub fn duplicate<'b>(&'a self, options: CopyOption) -> StringKey<'b> {
```

Comment:
@PhilippGackstatter Please confirm that this is an appropriate way to implement the copy flags in Rust (I basically took what you did for the lookup flags).

Also: Do you think, we should also add a binding for `keyCopy`? If so, how would we implement that?

Reply:
That looks good to me!

I honestly cannot remember the details of why I did not add a `keyCopy` binding. It probably had something to do with `keyDup` being very similar and easier to implement.
What are the use cases for `keyCopy` now? Do you have that written down somewhere? If they make sense in Rust then we could add a binding.